docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -152,6 +152,25 @@ You can perform these operations in bulk by multi-selecting rules, and then clic
 [role="screenshot"]
 image:images/bulk-mute-disable.png[The Manage rules button lets you mute/unmute, enable/disable, and delete in bulk,width=75%]
 
+[float]
+[[importing-and-exporting-rules]]
+=== Import and export rules
+
+To import and export rules, use the <<managing-saved-objects, Saved Objects Management UI>>.
+
+[NOTE]
+==============================================
+Some rule types cannot be exported through this interface:
+
+**Security rules** can be imported and exported using the {security-guide}/rules-ui-management.html#import-export-rules-ui[Security UI].
+
+**Stack monitoring rules** are <<kibana-alerts, automatically created>> for you and therefore cannot be managed via the Saved Objects Management UI.
+==============================================
+
+Rules are disabled on export. You are prompted to re-enable rule on successful import.
```

Comment:
Is there a step we should mention in scenarios they are using connectors with their rules?